CMakeLists.txt

```diff
@@ -532,8 +558,83 @@ endif()
 # Include the subdirectories
 add_subdirectory(g2o)
 
+# Create alias targets for easier usage when included as subdirectory
+# Core libraries
+if(TARGET core)
```

Comment:
I am not sure what is here best practice. We could also rename all the target in the respective sub-folders?

Reply:
I guess, I have a slight preference to provide the alias in the individual CMakeLists of the subfolder instead of here. This would also spare the `if(TARGET ...)` check.